examples/sveltekit/src/graphql/urql.error.ts

```diff
@@ -0,0 +1,13 @@
+import { errorExchange as urqlErrorExchange } from '@urql/svelte';
+
+export const errorExchange = () =>
+	urqlErrorExchange({
+		onError(error) {
+			if (error.graphQLErrors[0]) {
+				const message = error.graphQLErrors[0].message;
+				// @todo error
+			} else {
+				// @todo network error
+			}
+		}
+	});
```

Comment:
Imo, we can remove this. For example, I didn't use errorExchange in Nuxt example

Reply:
I would like to display notifications here, I may try I totally forgot

Reply:
Done